### PR TITLE
Release/10.0.0

### PR DESCRIPTION
**Pull request by petermason**
#### Description
Repace JS scripts in DomContext with the new Ad Library

Update id's to correct values for each ad type

Step 1a: Releases Ad LIb with only Admantix and Permutive without auctions and in current slots ('like-for-like')

#### Screenshots 

image

image

image

#### Checklist
 - [x] Unit tests
 - [ ] Updated E2E tests
 - [ ] Updated storybook
 - [x] Tested on iOS simulator
 - [x] Tested on Android emulator
 - [ ] Tested on Tablet

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "times-components-native",
   "private": true,
-  "version": "9.7.13",
+  "version": "10.0.0",
```
Comment (josholdham) [inline package.json:4]: 🤩

File: packages/ad/__tests__/ios/dom-context.ios.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import React from "react";
-import "../mocks";
+//import "../mocks";
```
Comment (josholdham) [inline packages/ad/__tests__/ios/dom-context.ios.js:2]: delete?

File: packages/ad/__tests__/ios/dom-context.ios.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+import { mockNativeModules } from "@times-components-native/mocks";
 import DOMContextNative from "../../src/dom-context";
 import TestRenderer from "react-test-renderer";
 import * as domContextUtils from "../../src/utils/dom-context-utils";
 
+mockNativeModules();
```
Comment (josholdham) [inline packages/ad/__tests__/ios/dom-context.ios.js:8]: is this needed as well as the beforeEach one?

File: packages/ad/src/dom-context.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ import {
 import { Viewport } from "@skele/components";
 import styles, { calculateViewportVisible } from "./styles/index";
 import { webviewEventCallbackSetupAsString } from "./utils/webview-event-callback-setup";
-import { AdInitAsString } from "./utils/ad-init";
+// import { AdInitAsString } from "./utils/ad-init";
```
Comment (josholdham) [inline packages/ad/src/dom-context.tsx:17]: delete?

File: packages/ad/__tests__/ios/dom-context.ios.js (hunk 2 above)
Comment (petermason) [reply packages/ad/__tests__/ios/dom-context.ios.js:2]: so this test actually needs to be re-written as part of the next phase (1b)

File: packages/ad/src/dom-context.tsx [hunk 5]
```diff
@@ -40,22 +40,23 @@ const articleEventEmitter = new NativeEventEmitter(ArticleEvents);
...
+    width = screenWidth,
+  } = props;
   const webViewRef = React.useRef<WebView>(null);
 
   const adHeight = heightProp
     ? heightProp + Number(styles.containerAdditionalHeight.height)
     : 0;
 
-  const [loaded, setLoaded] = useState(false);
+  const [loaded, setLoaded] = useState(true);
```
Comment (josholdham) [inline packages/ad/src/dom-context.tsx:59]: always hard to fully get a sense of things from a PR, but is this legitimate? just at first glance, it seems like a loaded state should be false to start with....

**Review by josholdham — APPROVED** [review]
Top stuff, looks pretty involved.

I've left a few relatively superficial comments just flagging tidy-ups, and one question about setting loading state to true initially, but it all looks good to me so approved

File: packages/ad/src/dom-context.tsx (hunk 4 above)
Comment (petermason) [reply packages/ad/src/dom-context.tsx]: done

File: packages/ad/src/ad.js [hunk 6]
```diff
@@ -112,23 +112,21 @@ export class AdBase extends PureComponent {
       width,
     } = this.props;
 
-    const { hasError, isAdReady, offline } = this.state;
+    const { hasError, offline } = this.state;
     const config = getSlotConfig(slotName, width || screenWidth, orientation);
     const data = determineData(config, this.props);
+    console.log("CONFIG", config);
```
[the remark this answers is not in the document]
Comment (petermason) [reply packages/ad/src/ad.js]: done

File: packages/ad/__tests__/ios/dom-context.ios.js (hunk 3 above)
Comment (petermason) [reply packages/ad/__tests__/ios/dom-context.ios.js:8]: I've disable this test for this version pending a rewrite on the next step

**Pull request closed without merging**